commit b5e7de51b53c211eadc6fe1bb0e1fecd1b33c194 (refs/changes/80/2380/1)
Author: Jan Wyszynski <jan.wyszynski@joyent.com>
Date:   2017-08-11T16:55:59+00:00 (2 years, 2 months ago)
    
    joyent/node-manta#322 test7 make target should be test8 to reflect latest
    version of node

diff --git a/Makefile b/Makefile
index e2e74d6..3b697b5 100644
--- a/Makefile
+++ b/Makefile
@@ -70,13 +70,13 @@ test: deps
 # with a node_modules/ for that version.
 #
 .PHONY: testall
-testall: test7 test6 test012 test010 test4
+testall: test8 test6 test012 test010 test4
 
-.PHONY: test7
-test7:
+.PHONY: test8
+test8:
 	@([[ -f test/node.paths ]] || (echo "no test/node.paths" && exit 1) \
-		&& echo "# Test with node `$(shell awk '/^7/ { print $$2 }' test/node.paths)/node --version`" \
-		&& PATH="$(shell awk '/^7/ { print $$2 }' test/node.paths):$(PATH)" \
+		&& echo "# Test with node `$(shell awk '/^8/ { print $$2 }' test/node.paths)/node --version`" \
+		&& PATH="$(shell awk '/^8/ { print $$2 }' test/node.paths):$(PATH)" \
 			NPM_CONFIG_LOGLEVEL=silent NPM_CONFIG_PROGRESS=false \
 			make clean test)
 
diff --git a/test/node.paths.example b/test/node.paths.example
index a80ab6e..c89df60 100644
--- a/test/node.paths.example
+++ b/test/node.paths.example
@@ -1,4 +1,4 @@
-7 ~/opt/node-7/bin
+8 ~/opt/node-8/bin
 6 ~/opt/node-6/bin
 4 ~/opt/node-4/bin
 0.12 ~/opt/node-0.12/bin
